d2l-input-checkbox.js

```diff
@@ -0,0 +1,201 @@
+/**
+`d2l-input-checkbox`
+Polymer-based web component for D2L checkboxes
+
+@demo demo/d2l-input-checkbox.html
+*/
+/*
+  FIXME(polymer-modulizer): the above comments were extracted
+  from HTML and may be out of place here. Review them and
+  then delete this comment!
+*/
+import '@polymer/polymer/polymer-legacy.js';
+
+import 'd2l-colors/d2l-colors.js';
+import 'd2l-typography/d2l-typography-shared-styles.js';
+import { Polymer } from '@polymer/polymer/lib/legacy/polymer-fn.js';
+// WORKAROUND: polymer-modulizer grabs non-existing Element export from polymer-element
+// TODO: Remove Element reference
+import { PolymerElement as Element } from '@polymer/polymer/polymer-element.js';
```

Comment:
Need to remove this and update the code that was referencing `Poilymer.Element`.